msi/tools/create_msi.ps1

```diff
@@ -7,5 +7,10 @@ candle.exe -ext WixUtilExtension.dll ./amazon-cloudwatch-agent.wxs
 light.exe -ext WixUtilExtension.dll ./amazon-cloudwatch-agent.wixobj
 
 # upload to s3
-aws s3 cp ./amazon-cloudwatch-agent.msi "s3://$bucket/integration-test/packaging/$version/amazon-cloudwatch-agent.msi"
-Write-Host "s3 for msi is s3://$bucket/integration-test/packaging/$version/amazon-cloudwatch-agent.msi"
\ No newline at end of file
+$bucketPath = "s3://$bucket/integration-test/packaging/$version/amazon-cloudwatch-agent.msi"
+if ($version -eq "") {
+    # This is a prod, nonprod, or nightly build.
+    $bucketPath = "s3://$bucket/$version/amazon-cloudwatch-agent.msi"
+}
+aws s3 cp ./amazon-cloudwatch-agent.msi $bucketPath
+Write-Host "s3 for msi is $bucketPath"
```

Comment:
Current GitHub workflow calls:
```
.\create_msi.ps1 ${{ github.sha }} ${{ secrets.S3_INTEGRATION_BUCKET }}
```
Which I will change to:
```
.\create_msi.ps1 "" ${{ secrets.S3_INTEGRATION_BUCKET }}
```

Reply:
Seems unlikely that we would need more.
The new caller can now control everything except the filename:
```
$bucketPath = "s3://$bucket/amazon-cloudwatch-agent.msi"
```

And instead of adding more functionality to this short script we could just add another script.

I'd prefer a few simple scripts to 1 complicated script with many options that does many different things.